stock_picking_eco_tag/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "LGPL-3",
-    "version": "13.0.1.0.0",
+    "version": "13.0.1.0.1",
```

Comment:
This is an improvement, not a fix, so I'd prefer this version bump:
```suggestion
    "version": "13.0.2.0.0",
```